crowbar_framework/app/barclamps/sqoop_barclamp.rb

```diff
@@ -15,17 +15,28 @@
 # limitations under the License.
 #
 
-barclamp:
-  name: hive
-  display: hive
-  version: 0
-  requires:
-    - hadoop
-  member:
-    - apachehadoop
-
-crowbar:
-  layout: 1
-  order: 400
-  run_order: 400
-  chef_order: 400
+class SqoopBarclamp < Crowbar::Registry::Barclamp
+  name "sqoop"
+  display "Sqoop"
+  description "Hadoop Sqoop component"
+
+  member [
+    "apachehadoop"
+  ]
+
+  requires [
+    "hadoop"
+  ]
+
+  listed true
+
+  layout 1
+  version 0
+  schema 3
+
+  order 500
+
+  nav(
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments.